.github/workflows/weekly_update.yaml

```diff
@@ -0,0 +1,47 @@
+name: Run Weekly Database Update Automation
+
+on:
+  schedule:
+    cron: '0 9 * * 6'  # 9am every Saturday
```

Comment:
This should probably run midnight on Friday, since that's when sets become legal - right? We should tie the scheduling to legality in Standard/release